src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin.java

```diff
@@ -0,0 +1,111 @@
+package edu.cmu.cs.cs214.rec10.plugin;
+
+import edu.cmu.cs.cs214.rec10.framework.core.GameFramework;
+import edu.cmu.cs.cs214.rec10.framework.core.GamePlugin;
+import edu.cmu.cs.cs214.rec10.framework.core.Player;
+
+import java.util.*;
+
+public class MoneyDoorPlugin implements GamePlugin {
+
+    private int[] arr = {-3, -2, -1, 0, 1, 2, 3, 4, 5, 6};
+    private int ONE = 1;
+    private int TEN = 10;
+
+    // The game framework
+    private GameFramework framework = null;
+
+    // The amount of money each player has
+    public HashMap<String, Integer> PlayerScores = new HashMap<String, Integer>();
+
+    public String getGameName() {
+        return new String("money door game");
```

Comment:
You don't need to make a new String because Strings are immutable. But nice try trying to use defensive copying!